src/app/layouts/tabbar/tabbar-sample-3/tabbar-sample-3.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="addrBar">
+    URL: 
+    <button igxButton="flat"
+        [igxToggleAction]="dropdown"
+        [igxDropDownItemNavigation]="dropdown">{{ currentDocumentLocation }}</button>
+</div>
+<igx-drop-down #dropdown [width]="'100px'" (onSelection)="onDropDownSelection($event)">
```

Comment:
We could use IgxSelectComponent instead of the button and dropdown.

Reply:
Addressed.